src/domain/users/users.repository.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { User } from '@/datasources/users/entities/users.entity.db';
+import { PostgresDatabaseModuleV2 } from '@/datasources/db/v2/postgres-database.module';
+
+@Module({
+  imports: [PostgresDatabaseModuleV2, TypeOrmModule.forFeature([User])],
+})
+export class UserRepositoryModule {}
```

Comment:
I would move this to a separate PR in which the repository is implemented, e.g. with methods for retrieving `User`s.

Reply:
This is needed to generate the migration file and create the database table. Would you suggest also doing that in a separate PR?

Reply:
Yes. I would scaffold the structure there as well, e.g. create a controller and domain module (that can be empty) as the repository would not be imported here, rather via the aforementioned.